### DISCUSSION TITLE
Question on usage for a nursery/tree farm business

### DISCUSSION THREAD

**Discussion by mfreeman451** [inventree/InvenTree#7565] [Q&A] [answered]
We're looking at adopting this to help run a small family nursery/tree farm business. We've figured out the pattern of you have an order from a supplier, you take delivery of that order and some how update your parts inventory, and then as you sell things you remove them from the inventory/database. But what about instances where we are maybe selling a service, like consulting, or dump/waste fees?

**Comment by SchrodingersGat** [comment]
These types of services can be put into the database as "virtual" parts - which do not have stock items

**Reply by miggland** [reply, marked as answer]
Virtual parts work with limitations.. See https://github.com/inventree/InvenTree/issues/7499

Other options: Using PO/SO extra lines. Making actual parts, with a fictitious stock count (ie, put 1000 "Waste fee" in your stock, then sell those..)

What solution works best depends on what exactly you're expecting to do with these lines within InvenTree..

**Reply by mfreeman451** [reply]
thanks for the help, appreciate it.
